components/common/Link.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ Used for styling the links in the navbar
+ Will display children
+*/
+
+import { Link as ChakraLink } from "@chakra-ui/react";
+import { css } from "@emotion/css";
+
+const Link = (props: any) => {
+  const getStyling = (variant: string | null) => {
+    switch (variant) {
+      case "navlink":
+        return css`
+          padding: 0.5rem 1rem;
```

Comment:
We should try to avoid writing custom CSS unless absolutely necessary.

Reply:
Also use px, mx convention whenever possible

Reply:
Done